### ISSUE TITLE
【企业微信支持】Can somebody help to add WeWorkChat Provider?

### ISSUE THREAD

**Issue by ywtoly** [hybridauth/hybridauth#1400]
## Bug, feature or question?
https://work.weixin.qq.com/
Enterprise WeChat is a similar enterprise instant messaging product to WeChat.

The OAuth2 principle should be similar to WeChat, and we hope to add relevant providers.

### Version and provider
src/Provider/WeChat.php

Thank you for your efforts!

**Comment by boHam-wang** [comment]
I have the same command, can anybody add the support of WeWork

**Comment by ywtoly** [comment]
> I have the same command, can anybody add the support of WeWork

Did you solve it?

**Comment by boHam-wang** [comment]
> > I have the same command, can anybody add the support of WeWork
> 
> Did you solve it?

Yes, I have solved the problem. I have developed the WeWork.php project code and added it to the provider directory, allowing for a smooth OAuth2 login through WeChat Enterprise.

**Comment by ywtoly** [comment]
Do you have a solution?

**Comment by ywtoly** [comment]
> > > I have the same command, can anybody add the support of WeWork
> > 
> > 
> > Did you solve it?
> 
> Yes, I have solved the problem. I have developed the WeWork.php project code and added it to the provider directory, allowing for a smooth OAuth2 login through WeChat Enterprise.

Has it been publicly released?
 Where?

**Comment by boHam-wang** [comment]
> > > > I have the same command, can anybody add the support of WeWork
> > > 
> > > 
> > > Did you solve it?
> > 
> > 
> > Yes, I have solved the problem. I have developed the WeWork.php project code and added it to the provider directory, allowing for a smooth OAuth2 login through WeChat Enterprise.
> 
> Has it been publicly released? Where?
This code has not been published yet because it is not perfect yet, but I plan to improve it and publish it to my github home page where I will post my project in due course. Maybe just a few days
